### ISSUE TITLE
The usage of 2 embedder losses (`E_loss_T0` and `E_loss`)

### ISSUE THREAD

**Issue by itakatz** [jsyoon0823/TimeGAN#81]
Hello and thank you very much for the code of the paper.

Training of the embedder is done in 2 of the training steps; once in the embedder training:
https://github.com/jsyoon0823/TimeGAN/blob/8f6181cb9b9d2fa0c930cd902411d9ac8a308e07/timegan.py#L234
and again in the joint training, when training the generator:
https://github.com/jsyoon0823/TimeGAN/blob/8f6181cb9b9d2fa0c930cd902411d9ac8a308e07/timegan.py#L270
However, in both lines the loss used is `E_loss_T0`. So the `E_loss`, which also depends on `G_loss_S`, is not used in the code. I would guess that it should be used in the joint training, is that the case?
(also, why does the feed dictionary includes the noise `Z`? I can't see any term which depends on `Z` in these losses)

**Comment by jsyoon0823** [comment]
Thanks. Yes. I think I need to remove E_loss part which is no longer used.
Yep. Z is also independent on E_loss_T0.
